scripts/covid19indiaORG/Covid19IndiaORG.py

```diff
@@ -0,0 +1,167 @@
+    if 'districts' in state_data:
+        # Get the all district data for the state.
+        districts_data = state_data['districts']
+    else:
+        districts_data = {}
+        district_table = pd.DataFrame({})
+
+    # For every district in the state, parse the timeseries.
+    for district_name, data in districts_data.items():
+        district_timeseries = _parse_timeseries(data)
+        # Store the data in a DataFrame, and transpose it.
+        # Transposing puts the dates in the Y-axis.
+        district_table = pd.DataFrame(district_timeseries).T
+
+        # If there is not wikidataId for the given district name, skip it.
+        if not (iso_code in DISTRICTS and district_name in DISTRICTS[iso_code]):
+            continue
+
+        # Get the WikidataId from the dictionary.
+        district_table['wikidataId'] = DISTRICTS[iso_code][district_name]
+
+    # Store the DataFrame for this district under the state table.
+    state_table = pd.concat([state_table, district_table])
+
+    if 'confirmed' in state_table and 'recovered' in state_table:
+        # Calculate the active cases = confirmed cases - recovered cases.
+        state_table['active'] = state_table['confirmed'] - state_table['recovered']
+
+    # Get rid of any rows that don't have a wikidataId.
+    state_table.dropna(subset=['wikidataId'], inplace=True)
+
+    # Find the columns in common between the table and the Config.DATA_TO_KEEP.
+    columns_to_keep = DATA_TO_KEEP.intersection(set(state_table.columns))
+
+    # Get rid of any rows that don't have at least one of the following.
+    state_table.dropna(subset=list(columns_to_keep), thresh=1, inplace=True)
+
+    # Only keep the following columns, the rest are not part of the import.
+    state_table = state_table[list(columns_to_keep ) + ['wikidataId']]
+
+    # Rename the Y-axis to be "date".
+    state_table.index.name = "date"
+
+    return state_table
+
+def Covid19IndiaORG(state_to_source: Dict[str, str] = None, output: str = "output.csv"):
+    """
+    For all states in the Config.STATES, query the API
+    and generate a DataFrame table for each state,
+    including the state's districts data, from the json response.
+    Combine each state DataFrame table into india_table.
+    Clean and then export india_table as a CSV file.
+    """
+
+    # If no data source is given, use the default Config.STATE_APIS
+    if not state_to_source:
+        state_to_source = STATE_APIS
+
+    # A DataFrame containing ALL merged state and district data.
+    india_table = pd.DataFrame({})
+
+    # For every state in state_to_source, download the data.
+    for iso_code, data_source in state_to_source.items():
+        # Download the json data for the specific state.
+        # Each state has its own API.
+        downloaded_data: Dict[str, Dict] = _download_data(data_source)
+
+        # If there is no wikidataId for the state, skip it.
+        if iso_code not in STATES:
```

Comment:
Can you tell me about how much this happens? Maybe also leave a note in README

Reply:
it doesn't really happen at all, it's just an edge case in case they add some other form of state and we don't have that state in the hashmap.